### PR TITLE
Address remaining feedback in lib/usd and plugin/adsk

### PR DESCRIPTION
**Pull request by HamedSabri-adsk**
Hey guys,

This PR fixes conditional orders in lib/usd and plugin/adsk.  

Only 6 files :)

### CODE REVIEW

File: lib/usd/hdMaya/adapters/proxyAdapter.cpp [hunk 1]
```diff
@@ -18,18 +18,18 @@
...
-#endif // WANT_UFE_BUILD
-
-#include <mayaUsd/nodes/proxyShapeBase.h>
-
 #include <hdMaya/adapters/adapterRegistry.h>
 #include <hdMaya/debugCodes.h>
 #include <hdMaya/delegates/proxyDelegate.h>
 #include <hdMaya/delegates/sceneDelegate.h>
 
+#include <mayaUsd/nodes/proxyShapeBase.h>
```
Comment (mattyjams) [inline lib/usd/hdMaya/adapters/proxyAdapter.cpp:26]: Combine into one group.

File: lib/usd/hdMaya/delegates/proxyDelegate.cpp [hunk 2]
```diff
@@ -31,17 +31,17 @@
 #include <pxr/base/tf/envSetting.h>
 #include <pxr/base/tf/type.h>
 
+#include <hdMaya/adapters/proxyAdapter.h>
+#include <hdMaya/debugCodes.h>
+#include <hdMaya/delegates/delegateRegistry.h>
+
+#include <mayaUsd/nodes/proxyShapeBase.h>
```
Comment (mattyjams) [inline lib/usd/hdMaya/delegates/proxyDelegate.cpp:38]: Combine into one group.

File: lib/usd/hdMaya/adapters/proxyAdapter.cpp (hunk 1 above)
Comment (HamedSabri-adsk) [reply lib/usd/hdMaya/adapters/proxyAdapter.cpp]: Thanks @mattyjams . Consider it done 6b66abaa8

File: lib/usd/hdMaya/delegates/proxyDelegate.cpp (hunk 2 above)
Comment (HamedSabri-adsk) [reply lib/usd/hdMaya/delegates/proxyDelegate.cpp]: Consider it done 6b66abaa8

**Comment by HamedSabri-adsk** [comment]
@mattyjams see c6eecc5 that addresses all your feedback in PART III.

**Pull request merged**